packages/calcite-components/src/components/color-picker/utils.ts

```diff
@@ -254,3 +255,26 @@ export function toNonAlphaMode(mode: SupportedMode): SupportedMode {
 
   return nonAlphaMode;
 }
+
+/**
+ * Returns adjusted left and top values of thumb node by substracting half of width/height of the thumb from computed left and top values.
+ * The current thumb node has 1px in width and height.
+ *
+ * @param left
+ * @param top
+ * @returns {[number, number]}
+ */
+export function getAdjustedLeftAndTop(left: number, top: number): [number, number] {
```

Comment:
Can you revisit the name to be more expressive? It's not clear to me by name alone what the context of left & top is.